data/abilities.js

```diff
@@ -1691,7 +1692,7 @@ let BattleAbilities = {
 	"lightmetal": {
 		shortDesc: "This Pokemon's weight is halved.",
 		onModifyWeight(weight) {
-			return weight / 2;
+			return weight >> 1;
```

Comment:
ಠ_ಠ

Reply:
I'm actually somewhat inclined to write this as `Math.trunc(weight / 2)` for readability.